<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCMainWindow</class>
 <widget class="QMainWindow" name="DCMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/tickets.ico</normaloff>:/images/tickets.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="lblCurrentFile">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>No File Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cmdDateLeft">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDateRange">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Date Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cmdDateRight">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>View Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboCalendarMode">
        <item>
         <property name="text">
          <string>Date Range</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Month</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="calendarLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="separator"/>
    <addaction name="actionLog_Off"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCustomer_List"/>
    <addaction name="actionTo_Do_List"/>
    <addaction name="actionAlerts"/>
    <addaction name="separator"/>
    <addaction name="actionStatus_Bar"/>
    <addaction name="actionTool_Bar"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTile_Vertically"/>
    <addaction name="actionTile_Horizontally"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAction_Types"/>
   </widget>
   <widget class="QMenu" name="menuAppointments">
    <property name="title">
     <string>Appointments</string>
    </property>
    <addaction name="actionNew_Appointment"/>
    <addaction name="actionShow_Appointment_List"/>
   </widget>
   <widget class="QMenu" name="menuPhone_Calls">
    <property name="title">
     <string>Phone Calls</string>
    </property>
    <addaction name="actionNew_Call"/>
   </widget>
   <widget class="QMenu" name="menuCheques">
    <property name="title">
     <string>Cheques</string>
    </property>
    <addaction name="actionNew_Cheque"/>
    <addaction name="actionView_Register"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuPhone_Calls"/>
   <addaction name="menuAppointments"/>
   <addaction name="menuCheques"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCustomer_List"/>
   <addaction name="actionTo_Do_List"/>
   <addaction name="actionAlerts"/>
   <addaction name="separator"/>
   <addaction name="actionNew_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockCustomerList">
   <property name="windowTitle">
    <string>Customers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="cmdRefreshAgeing">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="trvAgeing">
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockToDo">
   <property name="windowTitle">
    <string>My ToDo List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeWidget" name="trvToDoList">
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Start Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Customer</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Subject</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Alerts</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTreeWidget" name="treeWidget_3">
       <column>
        <property name="text">
         <string>Customer Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Current Balance</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Overdue Amount</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Search Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockCalendar">
   <property name="windowTitle">
    <string>Collection Calendar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ExtendedCalendarWidget" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Active Customers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTreeWidget" name="trvActiveCustomers">
       <column>
        <property name="text">
         <string>Company Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Pending Tasks</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Next Task</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total Balance</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTransactions">
   <property name="windowTitle">
    <string>Transactions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <item>
      <widget class="TransactionView" name="transWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockChequeRegister">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Cheque Register</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="cmdNewCheque">
         <property name="text">
          <string>New Cheque</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdDeposit">
         <property name="text">
          <string>Deposit Selected Cheque(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dtpChequeDate">
         <property name="displayFormat">
          <string>dd MMM yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CustomQueryView" name="lstCheques"/>
     </item>
     <item>
      <widget class="QTreeWidget" name="trvChequeList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Customer</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date Paid</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cheque Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Bank</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cheque No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLog_Off">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Remove-Male-User.ico</normaloff>:/images/Remove-Male-User.ico</iconset>
   </property>
   <property name="text">
    <string>Log Off</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/packing.ico</normaloff>:/images/packing.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCustomer_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/distributor-report.ico</normaloff>:/images/distributor-report.ico</iconset>
   </property>
   <property name="text">
    <string>Customer List</string>
   </property>
  </action>
  <action name="actionTo_Do_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/sign-up.ico</normaloff>:/images/sign-up.ico</iconset>
   </property>
   <property name="text">
    <string>To Do List</string>
   </property>
  </action>
  <action name="actionAlerts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Select-Language.ico</normaloff>:/images/Select-Language.ico</iconset>
   </property>
   <property name="text">
    <string>Alerts</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionTool_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tool Bar</string>
   </property>
  </action>
  <action name="actionTile_Vertically">
   <property name="text">
    <string>Tile Vertically</string>
   </property>
  </action>
  <action name="actionTile_Horizontally">
   <property name="text">
    <string>Tile Horizontally</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/add-event.ico</normaloff>:/images/add-event.ico</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="toolTip">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAction_Types">
   <property name="text">
    <string>Action Types</string>
   </property>
  </action>
  <action name="actionNew_Appointment">
   <property name="text">
    <string>New Appointment</string>
   </property>
  </action>
  <action name="actionShow_Appointment_List">
   <property name="text">
    <string>Show Appointment List</string>
   </property>
  </action>
  <action name="actionNew_Call">
   <property name="text">
    <string>Start New Actions</string>
   </property>
   <property name="toolTip">
    <string>New Action</string>
   </property>
  </action>
  <action name="actionNew_Cheque">
   <property name="text">
    <string>New Cheque</string>
   </property>
  </action>
  <action name="actionView_Register">
   <property name="text">
    <string>View Register</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
  <customwidget>
   <class>ExtendedCalendarWidget</class>
   <extends>QWidget</extends>
   <header>extendedcalendarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransactionView</class>
   <extends>QWidget</extends>
   <header>transactionview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCustomer_List</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockCustomerList</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockCustomerList</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionCustomer_List</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTool_Bar</sender>
   <signal>toggled(bool)</signal>
   <receiver>mainToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTo_Do_List</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockToDo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockToDo</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionTo_Do_List</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAlerts</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionAlerts</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatus_Bar</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
